bundles/org.openhab.binding.wifiled/README.md

```diff
@@ -42,24 +42,24 @@ You can choose between two drivers with different functionality:
 | CLASSIC |                       |         ✓         |        ✓        |
 | FADING  |            ✓          |                   |                 |
 
-While the CLASSIC driver let you choose and run device internal programs (e.g. alternating blue),
-all normal operations (turn on or off, switch color, set brightness, ...) are performed immediately
-and without any fading effect.
+While the CLASSIC driver lets you choose and run device internal programs (e.g. alternating blue),
+all normal operations (e.g. turn on or off, switch color, etc.) are performed immediately and
+without any fading effect.
 
-If you prefer to switch colors smoothly and to turn your light on and off by slightly rising/decreasing the brightness
-you should try the FADING driver.
-If selected you can also set the number of fading steps and the fading duration in the thing configuration.
-Note that each fading step will at least take 10 ms for being processed.
-This natural limit is given by the speed of the LED controller and your network speed.
-Thus, a color fading with a configured fading duration of 0s might still take some time (count with more than 1 second for 100 steps).
-IF the "FADING" driver is chosen the program channel and the programSpeed channel will not have any effect.
+If you prefer to switch colors smoothly and to turn your light on and off by slightly increasing/decreasing the brightness,
```

Comment:
Please put each sentence on a single line.

Reply:
Why?

Reply:
That makes it easier for review later on.

Reply:
We shouldn't write a README with one sentence per line any more than we should write a book with one sentence per line.

Reply:
The lines are auto-concatenated for displaying, so the user sees them as one paragraph even if they are separate lines in the file (like in HTML files).

Reply:
I've changed this back only because it was that way originally.